### PR TITLE
Drop support for Squeak 5.2

### PR DESCRIPTION
**Pull request by tom-richter**
Drop support for Squeak 5.2

### CODE REVIEW

File: packages/TelegramClient-Core.package/TCCGroupChat.class/instance/initializeFromChatEvent..st [hunk 1]
```diff
@@ -3,4 +3,6 @@ initializeFromChatEvent: anEvent
 
 	super initializeFromChatEvent: anEvent.
 	self groupId: ((anEvent at: 'type') at: self groupKey).
-	self isChannel: ((anEvent at: 'type') at: 'is_channel').
+	((anEvent at: 'type') at: 'is_channel') ifNotNil: [
```
Comment (phkeese) [inline packages/TelegramClient-Core.package/TCCGroupChat.class/instance/initializeFromChatEvent..st:6]: I would prefer `at: ifAbsent:` here. maybe

`self isChannel: ((anEvent at: 'type') at: 'is_channel' ifAbsent: false)`

File: packages/TelegramClient-UI.package/TCUPhotoMessage.class/instance/addPhoto.st [hunk 2]
```diff
@@ -2,11 +2,8 @@ constructing
 addPhoto
 
 	| photoForm |
-	photoForm := self messageModel core imageStore getFormFor: self messageModel fileId.
-
-	"To be compatible with Squeak 5.2 we can't use Form >> scaledToWidth.
-	photoForm := photoForm scaledToWidth: self class defaultWidth. is equivalent to the line below."
-	photoForm := photoForm magnify: photoForm boundingBox by: (self class defaultWidth / photoForm width) smoothing: 2.
+	photoForm := (self messageModel core imageStore getFormFor: self messageModel fileId).
```
Comment (phkeese) [inline packages/TelegramClient-UI.package/TCUPhotoMessage.class/instance/addPhoto.st:5]: parenthesis are not necessary

**Review by phkeese — CHANGES_REQUESTED** [review]
only one real request and some formatting, the rest looks good

**Comment by Storyxx** [comment]
maybe fix coveralls?

File: packages/TelegramClient-UI.package/TCUPhotoMessage.class/instance/addPhoto.st (hunk 2 above)
Comment (tom-richter) [reply packages/TelegramClient-UI.package/TCUPhotoMessage.class/instance/addPhoto.st]: removed

File: packages/TelegramClient-Core.package/TCCGroupChat.class/instance/initializeFromChatEvent..st (hunk 1 above)
Comment (tom-richter) [reply packages/TelegramClient-Core.package/TCCGroupChat.class/instance/initializeFromChatEvent..st]: removed this code as it was already fixed in a different commit

**Review by permler — APPROVED** [review]

**Pull request merged**